pandas/tests/indexes/test_multi.py

```diff
@@ -3307,3 +3307,39 @@ def test_duplicate_multiindex_labels(self):
         with pytest.raises(ValueError):
             ind.set_levels([['A', 'B', 'A', 'A', 'B'], [2, 1, 3, -2, 5]],
                            inplace=True)
+
+    def test_use_enum_in_multiindex(self):
+        # GH 21298
+        # Allow use of Enums as one of the factors in a MultiIndex.
+        from enum import Enum
+        from pandas.core.algorithms import take_1d
```

Comment:
By "top", @jreback meant "top of the file" 😄

Reply:
Hah whoops. done